modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/validation/rule/CommerceReturnReturnStatusObjectValidationRuleEngineImpl.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.commerce.internal.object.validation.rule;
+
+import com.liferay.object.constants.ObjectValidationRuleConstants;
+import com.liferay.object.scope.ObjectDefinitionScoped;
+import com.liferay.object.validation.rule.ObjectValidationRuleEngine;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Crescenzo Rega
+ */
+@Component(service = ObjectValidationRuleEngine.class)
+public class CommerceReturnReturnStatusObjectValidationRuleEngineImpl
```

Comment:
the name of this class sounds strange at least

Reply:
Hey @luca-pellizzon the approach was to use the first part of the name is the object name the second part is the field name, for this validator so we have the 'CommerceReturn' object and the 'ReturnStatus' field.
Let me know if we need to change the naming convention to something more readable